README.md

```diff
@@ -81,6 +81,50 @@ pnpm dev
 - @aLIEzsss4
 - @noyyyy
 
+## Contribute
+
+This repo contains all the source code of AutoChessia, and we welcome pull requests at any time. If you have any ideas for new features or modifications, we are happy to discuss them with you.
+
+Your feedback is very important to us. Whether it's code contributions or product suggestions, we sincerely want to hear your thoughts. Let's work together to make AutoChessia more powerful and user-friendly!
+
+If you encounter any issues during code contribution or usage, please feel free to reach out to us as well.
+
+Thank you for supporting AutoChessia! We look forward to having you on board!
+ [join our Discord]( https://discord.gg/Qget5JQHtr).
+
+### Local development setup
```

Comment:
There's some overlap here with `get start`. Merge them to chapter.

Reply:
i think get start doesn't conflict with contributions

Reply:
> i think get start doesn't conflict with contributions

@aLIEzsss4 Not agree. Writing document for how to start this project is a part of contribution guide. You can view the opensource guide website quickly before continue this pr. https://opensource.guide/